src/service/platform-impl.js

```diff
@@ -16,6 +16,12 @@
 
 import {registerServiceBuilder} from '../service';
 
+// Latest stable version numbers of browsers as of 12/3/2018
+const latestVersion = {
+  chrome: 71,
+  firefox: 64,
+  safari: 12,
+};
```

Comment:
I'll remove all changes to `platform-impl.js`. Pretty sure you're right because this changed the bundle size and failed the check.

The link above is for Chrome Apps. For Chrome I could look for the build number, which always increases, but the implementation would be different for Firefox. Is it ok to hardcode the version numbers in `_init_tests.js` for now?

Reply:
The situation I'm trying to avoid: when the next version is released, the current dev version number will become stable, and tests will suddenly stop being run on stable Chrome.

One thing to try: Debug this by printing the user agent and version name strings for Chrome, Chrome Dev, Firefox, and Firefox Dev, and see if the dev-ness of a browser version can be deduced from there.